sendsoknad-business/src/main/java/no/nav/sbl/dialogarena/soknadinnsending/business/service/soknadservice/SoknadDataFletter.java

```diff
@@ -364,6 +371,7 @@ public void sendSoknad(String behandlingsId, byte[] pdf, byte[] fullSoknad) {
             try {
                 List<Vedlegg> vedlegg = vedleggFraHenvendelsePopulator.hentVedleggOgKvittering(soknad);
                 innsendingService.sendSoknad(soknad, alternativeRepresentations, vedlegg, pdf, fullSoknad, fullSoknadId);
+                brukernotifikasjonService.cancelNotification(behandlingsId, behandlingsId,soknad.erEttersending(),soknad.getAktoerId());
```

Comment:
There is already code calling `brukernotifikasjon.newNotification()` in InnsendingService. We should put these two calls next to each other.